Essentials/src/com/earth2me/essentials/commands/Commandpweather.java

```diff
@@ -1,27 +1,21 @@
 package com.earth2me.essentials.commands;
 
-import com.google.common.collect.Lists;
-
 import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.IUser;
 import com.earth2me.essentials.User;
+import com.google.common.collect.Lists;
 import org.bukkit.Server;
 import org.bukkit.WeatherType;
-import org.bukkit.entity.Player;
 
 import java.util.*;
 
 import static com.earth2me.essentials.I18n.tl;
+import static com.earth2me.essentials.commands.Commandptime.getAliases;
```

Comment:
Big nope

Reply:
I noticed this after seeing the `protected` scope change earlier (hence why I asked why you did it). Now I see, but this isn't a good idea in my opinion.